pulse/service/src/test/java/gov/ca/emsa/pulse/service/security/BaseSecurityTest.java

```diff
@@ -96,11 +99,73 @@ public void restore() throws SQLException {
         /*
          * restore data set acf 1 liferay_state_id 1 liferay_state_id
          */
+    }
+
+    protected void setNullUser() {
+        JWTUserTestHelper.setNullUser();// the
```

Comment:
weird comment here

Reply:
refactor hangover. fixed locally, will get included in the next fix. thanks